code/API_definitions/location-retrieval.yaml

```diff
@@ -37,6 +37,10 @@ info:
 
     * **Max Age**: Maximum age of the location information which is accepted for the location retrieval (in seconds).
 
+      * Absence of maxAge means "any age" is acceptable for the client. In other words, this is like maxAge=infinite
+      * maxAge=0 means a fresh calculation is requested by the client.
+      If the system does not support it, or fresh location cannot be calculated at that time for any reason, the API response will be "unknown".

```

Comment:
I think this behavior can be helpful to add:
"In this case. the system may still return lastLocationTime, if available."